sdk/cdn/Azure.ResourceManager.Cdn/CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2024-04-23)
```

Comment:
Since you have updated the version in changelog, you need to update the version in [Azure.ResourceManager.Cdn.csproj](https://github.com/Azure/azure-sdk-for-net/blob/696fee322caab8080ccd08fbef3458aa6f28672a/sdk/cdn/Azure.ResourceManager.Cdn/src/Azure.ResourceManager.Cdn.csproj#L3) as well